lib/ack-tracker/batched_ack_tracker.c

```diff
@@ -125,6 +174,8 @@ _manage_msg_ack(AckTracker *s, LogMessage *msg, AckType ack_type)
 
   if (ack_type != AT_ABORTED)
     {
+      if (!iv_timer_registered(&self->batch_timer))
+        _start_batch_timer(self);
```

Comment:
To me this does not feel like a timeout, but a periodic forced ack. If I think about a timeout, I would expect the timer to be restarted, if a batch is acked.

Reply:
I think, you are right... will fix it soon.